vllm/model_executor/layers/activation.py

```diff
@@ -239,6 +239,35 @@ def extra_repr(self) -> str:
         return f'approximate={repr(self.approximate)}'
 
 
+@CustomOp.register("swiglu_oai")
+class SwiGLUOAI(CustomOp):
```

Comment:
Seems like this should be a `SwigluOAIAndMul`. Don't forget to put it in the registry at the bottom